modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -99,12 +99,15 @@ See https://coreos.github.io/afterburn/usage/initrd-network-cmdline/[the Afterbu
 
 WARNING: If you need networking to grab your Ignition config and your environment requires more complex networking than the default of DHCP to grab the Ignition config, then you'll need to use another method other than Ignition to configure the network.
 
-Networking configuration can be performed by writing out files described in an Ignition config. These are https://networkmanager.dev/docs/api/latest/nm-settings-keyfile.html[NetworkManager keyfiles] that are written to `/etc/NetworkManager/system-connections/` that tell NetworkManager what to do.
+There are to main methods for configuring networking with ignition 
+- By creating NetworkManager configuration files described in an Ignition config. These are https://networkmanager.dev/docs/api/latest/nm-settings-keyfile.html[NetworkManager keyfiles] that are written to `/etc/NetworkManager/system-connections/` that tell NetworkManager what to do.
+- By creating a nmstate desire state YAML(postfix .yml) files at /etc/nmstate following https://nmstate.io[NMstate syntax]. The applied network state file will be renamed with postfix .applied to prevent repeated applied on next run.
```

Comment:
I assume everywhere in this file we should be consisten with the capitalization of `NMstate`? Also a few wording suggestsion:

```suggestion
- By creating NMstate desired state YAML files with the `.yml` suffix under `/etc/nmstate/` following the https://nmstate.io[NMstate syntax]. The state files will be applied and then renamed with a `.applied` suffix to prevent being processed again.
```